plugins/vendor/services/ZoomVendorService.php

```diff
@@ -198,6 +206,10 @@ public function recordingCompleteAction()
 		{
 			throw new KalturaAPIException('Zoom Integration data Does Not Exist for current Partner');
 		}
+		if($zoomIntegration->getStatus()==VendorStatus::DISABLED)
```

Comment:
Maybe we should validated that status != ACTIVE since we don't want to process video for deleted vendor either